live_tests/test_data_manager_changes.py

```diff
@@ -0,0 +1,110 @@
+from gemd.entity.object import *
+import pytest
+from citrine.resources.dataset import Dataset
+from gemd.builders.impl import make_node
+from gemd.util.impl import flatten
+from citrine.seeding.find_or_create import find_or_create_team,find_or_create_dataset,find_or_create_project
+
+from citrine import Citrine
+
+TESTING_SCOPE = "pne-241-scope"
+
+def seed_registration_no_project(team):
+    dataset = find_or_create_dataset(dataset_collection=team.datasets, dataset_name="dataset 1", raise_error=True)
+    gemds = []
+    for i in range(1,5):
+        this_node = make_node(name = f"material {str(i)}", process_name= "fake process")
```

Comment:
Nitpick: in an f-string, `str` is called on variables automatically, so you don't need to do it yourself.

Reply:
Already addressed